process_labels/loading_funcs/utils.py

```diff
@@ -85,3 +85,19 @@ def export_date_from_row(row: pd.Series) -> datetime:
     overlapping_year = _overlapping_year(row.harvest_date, row.planting_date)
 
     return datetime(overlapping_year, EXPORT_END_MONTH, EXPORT_END_DAY)
+
+def _process_copernicusgeoglam(df:
+    geopandas.GeoDataFrame) -> geopandas.GeoDataFrame:
+    df = df.to_crs(LATLON_CRS)
+    
+    gdf = geopandas.GeoDataFrame(crs=LATLON_CRS, geometry=df.geometry)
+    gdf[RequiredColumns.LAT] = df.geometry.y
+    gdf[RequiredColumns.LON] = df.geometry.x
+    gdf[RequiredColumns.COLLECTION_DATE] = df.today
+    gdf[RequiredColumns.EXPORT_END_DATE] = datetime(2023, EXPORT_END_MONTH, EXPORT_END_DAY)
```

Comment:
Could the year be an argument to this function, since it seems plausible we will want to process this data for different years in the future? 2023 could be the default so that all the other code doesn't need to change:

```python
def _process_copernicusgeoglam(df: geopandas.GeoDataFrame, export_end_year: int = 2023):
```

Reply:
Sounds good!